dali/pipeline/operators/crop/bbox_crop.cc

```diff
@@ -14,33 +14,35 @@
 
 #include "dali/pipeline/operators/crop/bbox_crop.h"
 
-namespace dali
-{
+namespace dali {
 
 DALI_SCHEMA(BBoxCrop)
     .DocStr(
-        R"code(Perform a prospective crop to an image while keeping bounding boxes consistent.
-        Crop is provided as two Tensors: `Begin` which contains the starting coordinates for the `crop` in `(x,y)` format,
-        and 'Size' which contains the dimensions of the `crop` in `(w,h)` format. Bounding boxes are provided as a `(m*4)` Tensor,
+        R"code(Perform a prospective crop to an image while keeping bounding boxes consistent. Inputs must be supplied as two Tensors:
+        `Images` containing image data in NHWC format, and `BBoxes` containing bounding boxes represented as `[x,y,w,h]`.
```

Comment:
I wonder if we may need to support ltrb too, COCO reader does.

Reply:
I added a final commit. Bboxes are ltrb but crops are (xy) (wh), so partly a typo on my side

Reply:
I mean that COCO supports both types of the output ltrb and xywh. So we need to support both as well.